Task2/domain/src/main/java/com/epam/esm/repository/type/SearchTypeQuery.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.esm.repository.type;
+
+public enum SearchTypeQuery {
```

Comment:
Why do you use different approach: in repository you have used private static final String(s), and here to create pre-query you use ENUM.
It is mostly better to apply only one approach.